db/iaito

```diff
@@ -5,11 +5,18 @@ R2PM_DESC "[gui] C++/Qt GUI for radare2"
 
 R2PM_INSTALL() {
 	cd src || exit 1
-	export PATH="~/Qt/5.6/clang_64/bin:$PATH"
+	export PATH="$HOME/Qt/5.6/clang_64/bin/:$HOME/Qt/5.6/gcc_64/bin/:$PATH"
 	qmake -makefile iaito.pro || exit 1
 	${MAKE} -j6 || exit 1
-	SRC="${PWD}/iaito.app/Contents/MacOS/iaito"
-	${R2PM_SUDO} ln -fs ${SRC} ${R2PM_PREFIX}/bin/iaito || exit 1
+        mkdir -p ${R2PM_PREFIX}/bin/
+	if [ "`uname`" = 'Linux' ]
```

Comment:
i think this will be the same for *BSD, so just put the other block under 'Darwin' instead of this one only for Linux